<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FahrzeugNeu</class>
 <widget class="QWidget" name="FahrzeugNeu">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neues Fahrzeug</string>
  </property>
  <widget class="QLineEdit" name="lineEditPreisProTag">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBezeichnung">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Typ:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Preis pro Tag:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKennzeichen">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kennzeichen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTyp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBeenden">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Bezeichnung:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditBezeichnung</tabstop>
  <tabstop>lineEditKennzeichen</tabstop>
  <tabstop>lineEditPreisProTag</tabstop>
  <tabstop>lineEditTyp</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonBeenden</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
